abi-gen/package.json

```diff
@@ -93,6 +93,7 @@
         "@types/glob": "5.0.35",
         "@types/mkdirp": "^0.5.2",
         "@types/mocha": "^5.2.7",
+        "@types/sinon": "^2.2.2",
```

Comment:
an addition? 🤨

Reply:
Oh even better

Reply:
It was used in `abi-gen`, but wasn't there -- it was silently relying on the dep from `subproviders`. Which didn't use it. So when I removed it from `subproviders` `abi-gen` broke.